packages/rspack/tests/hotCases/plugins/banner/test.filter.js

```diff
@@ -0,0 +1 @@
+module.exports = () => false;
```

Comment:
This case wrote by myself use global directly and will always need to use vm.runInThisContext, disable it and find a better way in other pull request